### PR TITLE
Luis commands stdin enabling

### PR DESCRIPTION
**Pull request by munozemilio**

### CODE REVIEW

File: packages/luis/src/parser/converters/lutoluisconverter.js [hunk 1]
```diff
@@ -7,10 +7,16 @@ const exception = require('./../lufile/classes/exception')
 const LUISObjNameEnum = require('./../lufile/enums/luisobjenum')
 
 module.exports = {
-    parseLuToLuis: async function(files, verbose, luis_culture) {
+    /**
+     * Parses a list of luObject into a LUIS JSON
```
Comment (vishwacsena) [inline packages/luis/src/parser/converters/lutoluisconverter.js]: Doc needs fix up.

File: packages/luis/src/parser/luisfile/parseLuisFile.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ module.exports = {
         try {
             LUISJSON = await JSON.parse(LUISFileContent);
         } catch (err) {
-            throw (new exception(retCode.errorCode.INVALID_INPUT_FILE, 'Sorry, error parsing file as LUIS JSON: ' + file));
+            throw (new exception(retCode.errorCode.INVALID_INPUT_FILE, 'Sorry, error parsing file as LUIS JSON'));
```
Comment (vishwacsena) [inline packages/luis/src/parser/luisfile/parseLuisFile.js]: Should this instead read as 'Sorry, error parsing content as LUIS JSON'?

File: packages/luis/src/parser/converters/lutoluisconverter.js [hunk 3]
```diff
@@ -7,10 +7,16 @@ const exception = require('./../lufile/classes/exception')
...
+     * @param {luObject []} luArray luObject list to be parsed
+     * @returns {LUIS} Collated LUIS json contents
+     * @throws {exception} Throws on errors. exception object includes errCode and text. 
+     */
+    parseLuToLuis: async function(luArray, verbose, luis_culture, luSearchFn) {
         try {
             // Extract all lu files and merge all into and object
-            let allParsedContent = await lu.mergeAndResolveReferences(files, verbose, luis_culture)
+            let allParsedContent = await lu.mergeAndResolveReferences(luArray, verbose, luis_culture, luSearchFn)
```
Comment (vishwacsena) [inline packages/luis/src/parser/converters/lutoluisconverter.js]: Can we not default the call back function to be the disk I/O based one we have baked in?

File: packages/luis/src/parser/converters/lumerger.js [hunk 4]
```diff
@@ -16,13 +16,13 @@ const luisJSON = require('./../luisfile/parseLuisFile');
...
     mergeAndResolveReferences: async function (luObjArray, verbose, luis_culture, luSearchFn){
-        let allParsedContent = await buildLuObject(luObjArray, verbose, luis_culture, luSearchFn)
+        let allParsedContent = await buildJsonObject(luObjArray, verbose, luis_culture, luSearchFn)
         await resolveReferencesInUtterances(allParsedContent)
         return allParsedContent
     }
 }
 
-const buildLuObject = async function(luObjArray, log, luis_culture, luSearchFn = findLuFilesInDir){
+const buildJsonObject = async function(luObjArray, log, luis_culture, luSearchFn = findLuFilesInDir){
```
Comment (vishwacsena) [inline packages/luis/src/parser/converters/lumerger.js]: Minor nit - Consider renaming to buildLUISJsonObject or `buildLuJsonObject` - given we have LU, QnA, QnA alternations content in there, emphasizing `LU` might improve readability.

File: packages/luis/src/parser/converters/lutoluisconverter.js (hunk 1 above)
Comment (munozemilio) [reply packages/luis/src/parser/converters/lutoluisconverter.js]: ??

**Pull request merged**